docs/install/macos.md

```diff
@@ -0,0 +1,35 @@
+# macOS
+
+You can use [Homebrew](https://brew.sh) to install the `coder` command. Homebrew
```

Comment:
I think this whole doc should go in the system packages section.

Reply:
I can see both sides. I think it's nice to give macOS parity with Windows by giving users of them each an obvious page to look at.